core/src/smartcontracts/isi/mod.rs

```diff
@@ -270,15 +303,14 @@ mod tests {
         let world = World::with([], PeersIds::new());
         let query_handle = LiveQueryStore::test().start();
         let state = State::new(world, kura.clone(), query_handle);
-        let genesis_account_id = AccountId::from_str("genesis@genesis")?;
-        let account_id = AccountId::from_str("alice@wonderland")?;
-        let (public_key, _) = KeyPair::random().into_parts();
+        let genesis_account_id: AccountId = "genesis@genesis".parse_alias();
```

Comment:
Maybe a form of `AccountId::from_sample_alias("genesis@genesis")` would also work, and will reduce the diff. (current is also not bad).